cmd/nerdctl/compose/compose_up.go

```diff
@@ -102,6 +104,17 @@ func composeUpAction(cmd *cobra.Command, services []string) error {
 	if err != nil {
 		return err
 	}
+	forceRecreate, err := cmd.Flags().GetBool("force-recreate")
+	if err != nil {
+		return err
+	}
+	noRecreate, err := cmd.Flags().GetBool("no-recreate")
+	if err != nil {
+		return nil
```

Comment:
return error here?

Reply:
I copied this from `compose_create.go`, I think they should probably both return the errs, good catch